interfaces/repo.go

```diff
@@ -736,9 +747,11 @@ func (r *Repository) AutoConnectCandidates(plugSnapName, plugName string) []*Slo
 	if plug == nil {
 		return nil
 	}
-	if r.ifaces[plug.Interface].AutoConnect() == false {
+
+	if (r.ifaces[plug.Interface].AutoConnect() == false) && !(isSpecialAutoConnectSnap(plug.Snap)) {
```

Comment:
It's a bit awkward to do that at this level. It's ignoring what the interface is entirely.

I'll complement this comment below.